pyproject.toml

```diff
@@ -102,3 +103,56 @@ module = [
   "filter_availability",
   "matplotlib.*"
 ]
+
+[tool.cibuildwheel]
+build-verbosity = 1
+build-frontend = "build"
+skip = [
+  "*-musllinux*",
+  "cp314t-*",
+]
+test-extras = "tests"
+test-sources = [
+  "test",
+  "pyproject.toml"
+]
+test-command = [
+    '''python -c "import netCDF4; print(f'netCDF4 v{netCDF4.__version__}')"''',
+    "pytest -s -rxs -v test",
+]
+manylinux-x86_64-image = "ghcr.io/ocefpaf/manylinux_2_28_x86_64-netcdf"
+manylinux-aarch64-image = "ghcr.io/ocefpaf/manylinux_2_28_aarch64-netcdf"
+environment = {NETCDF4_LIMITED_API="1"}
+
+[tool.cibuildwheel.macos]
+before-build = "brew install hdf5 netcdf"
+
+[[tool.cibuildwheel.overrides]]
+select = "*-macosx_*_x86_64*"
+inherit.environment = "append"
+environment = {MACOSX_DEPLOYMENT_TARGET="13.0"}
+
+[[tool.cibuildwheel.overrides]]
+select = "*-macosx_*_arm64*"
+inherit.environment = "append"
+environment = {MACOSX_DEPLOYMENT_TARGET="14.0"}
+
+[tool.cibuildwheel.windows]
+before-build = "python -m pip install delvewheel"
+repair-wheel-command = [
+  "delvewheel show {wheel}",
+  "delvewheel repair -w {dest_dir} {wheel}",
+]
+
+[[tool.cibuildwheel.overrides]]
+select = "*-win_*"
+inherit.environment = "append"
+environment = {HDF5_DIR='C:\\\\Users\\runneradmin\\micromamba\\envs\\build\\Library', netCDF4_DIR='C:\\\\Users\\runneradmin\\micromamba\\envs\\build\\Library', PATH='C:\\\\Users\\runneradmin\\micromamba\\envs\\build\\Library\\bin;${PATH}' }
+
+[[tool.cibuilwheel.overrides]]
+select = "*-win_*"
+inherit.skip = "append"
+skip = [
+  # cannot build cftime for this target (missing a wheel at the time of writing)
+  "cp314",
+]
```

Comment:
this doesn't seem to work either. Could this be a bug that cibuildwheel ignores it ?

Reply:
reported as https://github.com/pypa/cibuildwheel/issues/2596